libs/ledger-live-common/src/deviceSDK/hooks/useBatteryStatuses.ts

```diff
@@ -41,7 +41,8 @@ export const useBatteryStatuses = ({
     useState<GetBatteryStatusesActionState>(initialState);
   const [requestCompleted, setRequestCompleted] = useState<boolean>(false);
   const [nonce, setNonce] = useState(0);
-  const [cancelRequest, setCancelRequest] = useState<() => void>(() => {});
+  const [cancelRequest, setCancelRequest] = useState(() => () => {}); // when passing a function to useState, the function is used as an initializer, i.e its return value will be the initial state value, cf. https://react.dev/reference/react/useState#parameters
```

Comment:
It is possible to split this comment into multiple lines to avoid exceeding `100` characters ? `100` characters is the value defined in the `.prettierrc` file